scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -31,8 +34,11 @@ remove_dotnet_pkgs
 [ "$?" -ne 0 ] && echo "Failed to remove dotnet packages." && exit 1
 
 echo "Deleting install root - $dotnet_install_root"
-rm -rf $dotnet_install_location
+rm -rf $dotnet_install_root
 rm -f $dotnet_path_file
 
+echo "Deleting NuGet package location"
```

Comment:
Nuking anything inside a user's home directory sounds really dangerous to me. If we're really doing this, what about other users' home directories?